analyzer/src/main/java/org/jboss/gm/analyzer/alignment/AlignmentTask.java

```diff
@@ -273,15 +275,26 @@ private void align(Configuration configuration, ManipulationCache cache, Manipul
         logger.info("Completed scanning {} projects; now processing for exclusions/REST/overrides...",
                 cache.getDependencies().size());
         final List<ProjectVersionRef> allDeps = cache.getDependencies().values().stream()
-                .flatMap(m -> m.values().stream()).collect(Collectors.toList());
+                .flatMap(m -> m.values().stream()).sorted().distinct().collect(Collectors.toList());
 
-        final AlignmentService alignmentService = AlignmentServiceFactory
-                .getAlignmentService(configuration, cache.getDependencies().keySet());
+        final AlignmentService alignmentService = new DAAlignmentService(configuration);
+        final List<AlignmentService.Manipulator> manipulators = Stream
+                .of(new UpdateProjectVersionCustomizer(configuration, rootProject),
+                        new DependencyOverrideCustomizer(configuration, rootProject.getAllprojects()))
+                .sorted(comparingInt(AlignmentService.Manipulator::order))
+                .collect(Collectors.toList());
 
         final Response alignmentResponse = alignmentService.align(
                 new AlignmentService.Request(cache.getProjectVersionRefs(configuration.versionSuffixSnapshot()),
                         allDeps));
 
+        // Apply the current manipulators (DependencyOverride and UpdateProjectVersion)
+        // While they do support order, it's not hugely important given we only have two
+        // currently.
```

Comment:
You sort the manipulators on line 281, but this comment seems to imply they are unsorted.

Reply:
As those two manipulators don't affect each other the order currently isn't important. I could have stripped the ordering out but left it in for any future expansion.